pandas/tests/extension/test_arrow.py

```diff
@@ -682,62 +682,9 @@ def test_view(self, data):
 
 
 class TestBaseMissing(base.BaseMissingTests):
-    def test_fillna_limit_pad(self, data_missing, using_array_manager, request):
-        if using_array_manager and pa.types.is_duration(
-            data_missing.dtype.pyarrow_dtype
-        ):
-            request.node.add_marker(
-                pytest.mark.xfail(
-                    reason="Checking ndim when using arraymanager with duration type"
-                )
-            )
-        super().test_fillna_limit_pad(data_missing)
-
-    def test_fillna_limit_backfill(self, data_missing, using_array_manager, request):
-        if using_array_manager and pa.types.is_duration(
-            data_missing.dtype.pyarrow_dtype
-        ):
-            request.node.add_marker(
-                pytest.mark.xfail(
-                    reason="Checking ndim when using arraymanager with duration type"
-                )
-            )
-        super().test_fillna_limit_backfill(data_missing)
-
-    def test_fillna_series(self, data_missing, using_array_manager, request):
-        if using_array_manager and pa.types.is_duration(
-            data_missing.dtype.pyarrow_dtype
-        ):
-            request.node.add_marker(
-                pytest.mark.xfail(
-                    reason="Checking ndim when using arraymanager with duration type"
-                )
-            )
-        super().test_fillna_series(data_missing)
-
-    def test_fillna_series_method(
-        self, data_missing, fillna_method, using_array_manager, request
-    ):
-        if using_array_manager and pa.types.is_duration(
-            data_missing.dtype.pyarrow_dtype
-        ):
-            request.node.add_marker(
-                pytest.mark.xfail(
-                    reason="Checking ndim when using arraymanager with duration type"
-                )
-            )
-        super().test_fillna_series_method(data_missing, fillna_method)
-
-    def test_fillna_frame(self, data_missing, using_array_manager, request):
-        if using_array_manager and pa.types.is_duration(
-            data_missing.dtype.pyarrow_dtype
-        ):
-            request.node.add_marker(
-                pytest.mark.xfail(
-                    reason="Checking ndim when using arraymanager with duration type"
-                )
-            )
-        super().test_fillna_frame(data_missing)
+    @pytest.mark.filterwarnings("ignore:Falling back:pandas.errors.PerformanceWarning")
+    def test_dropna_array(self, data_missing):
```

Comment:
Going to backport this test change in this section to mark backporting easier for arrow PRs